penguin/web/routes.py

```
@@ -121,7 +131,9 @@ def _format_error_response(error: Exception, status_code: int = 500) -> HTTPExce
         )
 
 
-def _serialize_task_payload(task: Any, *, include_metadata: bool = True) -> Dict[str, Any]:
+def _serialize_task_payload(
+    task: Any, *, include_metadata: bool = True
```

Ruff (0.15.10)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `task`

(ANN401)